### PR TITLE
[Federation] Update the release location for kubefed

### PR DESCRIPTION
**Pull request by irfanurrehman**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.10 Features: set Milestone to 1.10 and Base Branch to release-1.10
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7429)

### CODE REVIEW

**Comment by heckj** [comment]
/assign @csbell @quinton-hoole

👋 could use a technical accuracy check on this please. If it's good, would you mind adding an /lgtm comment onto this PR?

**Comment by irfanurrehman** [comment]
@csbell, @quinton-hoole, can you please take a look at this.
@shashidharatd can also please comment on the technical accuracy of this PR.

File: docs/tasks/federation/set-up-cluster-federation-kubefed.md [hunk 1]
```diff
@@ -24,52 +24,47 @@ This guide assumes that you have a running Kubernetes cluster. Please
...
 
 ```shell
 sudo cp kubernetes/client/bin/kubefed /usr/local/bin
 sudo chmod +x /usr/local/bin/kubefed
-sudo cp kubernetes/client/bin/kubectl /usr/local/bin
-sudo chmod +x /usr/local/bin/kubectl
 ```
 
-### Install with snap on Ubuntu
+### For k8s versions 1.9.x:
```
Comment (csbell) [inline docs/tasks/federation/set-up-cluster-federation-kubefed.md]: For k8s versions 1.9.x and above?

Do you plan to edit this file for every release (1.10 is around the corner)

Comment (csbell) [inline docs/tasks/federation/set-up-cluster-federation-kubefed.md]: Might be useful to explain why RELEASE_VERSION doesn't work with 1.9 and newer -- move out of tree and that the kubefed binary is no longer distributed with the kubernetes-client package.

**Comment by irfanurrehman** [comment]
@csbell thanks for the review. Have updated as per your comments. PTAL.

File: docs/tasks/federation/set-up-cluster-federation-kubefed.md [hunk 2]
```diff
@@ -24,52 +24,57 @@ This guide assumes that you have a running Kubernetes cluster. Please
...
+Download the client tarball corresponding to the particular release and 
+extract the binaries in the tarball:
 
-```shell
-# Linux
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/kubernetes-client-linux-amd64.tar.gz
-tar -xzvf kubernetes-client-linux-amd64.tar.gz
+> Note that until kubernetes versions `1.8.x` the federation project was 
+maintained as part of [core kubernetes repo](https://github.com/kubernetes/kubernetes).
+At some point between kuberntes releases `1.8.0` and `1.9.0`, it moved into
```
Comment (tengqm) [inline docs/tasks/federation/set-up-cluster-federation-kubefed.md]: kuberntes typo

File: docs/tasks/federation/set-up-cluster-federation-kubefed.md [hunk 3]
```diff
@@ -24,52 +24,57 @@ This guide assumes that you have a running Kubernetes cluster. Please
...
+sudo cp federation/client/bin/kubefed /usr/local/bin
+sudo chmod +x /usr/local/bin/kubefed
+```
 
-       sudo snap install kubefed --classic
+### Install kubectl
 
-2. Run [`kubefed version`](/docs/admin/kubefed_version/) to verify that the version you've installed is sufficiently up-to-date.
+You can install a matching version of kubectl using the instructions on 
+the  [kubectl install plage](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```
Comment (tengqm) [inline docs/tasks/federation/set-up-cluster-federation-kubefed.md]: plage typo

File: docs/tasks/federation/set-up-cluster-federation-kubefed.md (hunk 2 above)
Comment (irfanurrehman) [reply docs/tasks/federation/set-up-cluster-federation-kubefed.md]: thanks @tengqm for checking this. Updated!

File: docs/tasks/federation/set-up-cluster-federation-kubefed.md (hunk 3 above)
Comment (irfanurrehman) [reply docs/tasks/federation/set-up-cluster-federation-kubefed.md]: Updated!

**Pull request merged**